packages/devtools_app/lib/src/screens/memory/panes/control/settings_dialog.dart

```diff
@@ -10,6 +10,12 @@ import '../../../../shared/globals.dart';
 import '../../../../shared/theme.dart';
 import '../../memory_controller.dart';
 
+/// The dialog keys for testing purposes.
+class MemorySettingDialogKeys {
```

Comment:
these don't need to be in a class. There should be a lint triggering for this to avoid classes with only static members

Reply:
it is ok for enum-like classes: 
```
However, this isn’t a hard rule. With constants and enum-like types,
it may be natural to group them in a class:

class Color {
  static const red = '#f00';
  static const green = '#0f0';
  static const blue = '#00f';
  static const black = '#000';
  static const white = '#fff';
}
```

https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

and there is no lint

Reply:
The lint is here: https://dart-lang.github.io/linter/lints/avoid_classes_with_only_static_members.html This used to be enabled for our repo - I'm surprised this isn't triggering analysis warning. 

But okay to leave. In general, we do not want to get in the habit of adding keys just for testing, because keys can have a performance impact and other side effects - they do more than just give the widget an "id", so it's good to take caution. In some cases, it's okay to add them for testing if using finders to find a widget in tests is overly complicated without a key, but in general it's good to avoid.